pkgs/top-level/all-packages.nix

```diff
@@ -35798,13 +35798,15 @@ with pkgs;
 
   scaleway-cli = callPackage ../tools/admin/scaleway-cli { };
 
+  bean-add = callPackage ../applications/office/beancount/bean-add.nix { };
+
   beancount = with python3.pkgs; toPythonApplication beancount;
 
   beancount-black = with python3.pkgs; toPythonApplication beancount-black;
 
   beanhub-cli = with python3.pkgs; toPythonApplication beanhub-cli;
 
-  bean-add = callPackage ../applications/office/beancount/bean-add.nix { };
+  beanprice = with python3.pkgs; toPythonApplication beanprice;
```

Comment:
Optionally:

1. Create a file `pkgs/by-name/be/beanprice/package.nix` with the contents below:

```nix
{ python3Packages }:

with python3Packages; toPythonApplication beanprice
```
(run the formatter, as always)

2. Delete the line above:
```suggestion
```